infogami/infobase/dbstore.py

```diff
@@ -612,6 +613,13 @@ def find_user(self, email):
         thing_id = d and d[0].thing_id or None
         return thing_id and self.get_metadata_from_id(thing_id).key
 
+    def check_datetime(self, obj):
```

Comment:
I don't think this should be a method inside the object. It's just a helper function not specific to this object. So, please make this function global.

A bit of documentation could go a long way :)
```
"""Helper function to serialize ``datetime.datetime`` object to JSON compatible.
  
 This is only to be used in the ``default`` argument to ``json`` module
 function/class. It can be extended to serialize other types of objects as per
 the requirement."""
```